tasks/managed/update-package-collection/update-package-collection.yaml

```diff
@@ -452,8 +497,13 @@ spec:
           # Extract the NVR (Name-Version-Release) from the SRPM file
           package_nvr=$(basename "$srpm_file" .src.rpm)
 
-          # Display the package NVR
-          echo "Package NVR: $package_nvr"
+          # Add the postfix ",draft_${build_id}" if it is a import draft build
+          if [[ "$KOJI_IMPORT_DRAFT" != "false" && "$PUSH_TYPE" != "promote" ]]; then
```

Comment:
```suggestion
          if [[ "$KOJI_IMPORT_DRAFT" == "true" ]]; then
```

Maybe we don't need the promote. Just check the if [[ "$KOJI_IMPORT_DRAFT" != "true" ]]; as what you deleted in this PR upon?